.eslintrc

```diff
@@ -200,7 +200,17 @@
     },
     {
       "files": [
-        "**/karma/**/*.js",
+        "**/karma/**/*.js"
+      ],
+      "plugins": [
+        "header"
+      ],
+      "rules": {
+        "no-undef": "off"
```

Comment:
Any reason why?

Reply:
These are files specific to Karma configuration and there are a number of Jasmine and Puppeteer globals and I figured it's ok to just turn them off for now.

Reply:
Let's use

```js
{
  "env": {
    "jasmine": true
  }
}
```

instead

Reply:
That's a lot better! Done.